os/board/rtl8721csm/src/component/os/os_dep/include/osdep_service.h

```diff
@@ -1,5 +1,5 @@
 /******************************************************************************
- * Copyright (c) 2013-2016 Realtek Semiconductor Corp.
+ * Copyright (c) 2020, Realtek Semiconductor Corp.

```

Comment:
This is start year of license. So you don't need to change.
But Please add Samsung license.

Reply:
Start year have been revert. Because this is arch and board folder so we decide to just stick with Realtek license.
Thanks